### ISSUE TITLE
The latest version cannot integrated with screen readers? 

### ISSUE THREAD

**Issue by NiKola-UE** [espeak-ng/espeak-ng#1903]
First, I apologize if this issue has already been asked. 

What I like about the latest version is that almost all voices and languages ​​are installed automatically, so you don't have to add them manually, which I don't like. However, although there are no problems with the installation even after it, I can't find it in screen readers JAWS and NVDA when I change the speech synthesizers in them in my Windows 11. Of course, NVDA already has built-in ESpeak, but I'm not talking about that here, but about the ESpeak application for Windows of already installed speech that works separately and independently of NVDA's integrated ESpeak; which was flawless in earlier versions, although the installation was much more complicated and difficult. If ESpeak still supports Sapi5, it's strange that it doesn't integrate with it. 

The Command Prompt-based text input app works and recognizes the language of the input text, but only one intermittent male voice is heard and I can't find any boxes anywhere to change the language, voices, speed, pitch, modulation, etc. 

Thank you in advanced.

**Comment by NiKola-UE** [comment]
@FelipeZanabria, please can you to help me for this? Well, the title maibe is not a correct, but the description of the problem is very clear.

**Comment by FelipeZanabria** [comment]
ESpeak-ng is not visible to screen readers because it is not supported by sapi5.
ESpeak-ng was restructured, so support has not yet been implemented.
You have to google espeak-ng command line options. There you can select the voice, speed, tone, volume, and even save the generated audio in a .wav file.
To be exact, in the old espeak for sapi5, all the files are installed on the hard drive. The dialogue to choose the voices is to add to the list of voices for sapi5.

**Comment by NiKola-UE** [comment]
Dear Felipe, 






Thank you for reacting, although I have already annoyed you with my comments before, but that was only due to my ignorance. 


Yes, I expressed myself incorrectly. Actually I don't mean Sapi 5, but integration with screen readers. After installing ESpeak-NG, I can't find it listed among synths at all in either JAWS and NVDA. With earlier versions of old ESpeak, there was no problem with this, although voices and languages ​​had to be entered separately and manually. There is no such thing here because everything was installed automatically, but I don't know how to use it like all other synthesizers. Should the voices be downloaded and installed separately here as well, or is something else being done? 








P.S. Croatian MBrola voice which I mentioned early was used (maibe only) for Simbian's Mobile Speak, but I really don't known how.

**Comment by NiKola-UE** [comment]
I apologize, I overlooked that there are already several issues like this. But it would be nice if someone answered at least one precisely so that there would be no more such and similar ambiguities.
